src/style.css

```diff
@@ -1,3 +1,95 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: gray;
+}
+
+ul {
+  list-style: none;
```

Comment:
Don't use tag names for styling. Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.